.github/workflows/ansible-test.yml

```diff
@@ -77,8 +77,8 @@ jobs:
     - name: Install the collection tarball
       run: ansible-galaxy collection install .cache/collection-tarballs/*.tar.gz
 
-    - name: Install galaxy-importer
-      run: pip install galaxy-importer
+    - name: Install galaxy-importer (v0.4.20)
```

Comment:
I had to pin to this version because the latest versions (0.4.21) released last week prints out quite a few unrelated warnings and fails the CI.

Reply:
Should we create a issue for this to update CI?

Reply:
Yep, done - issue #639